bph_pal/useful_commands.md

```diff
@@ -0,0 +1,27 @@
+Testing a package locally
+```
+pip3 uninstall -y bph_pal
+rm -rf dist
+python3 setup.py sdist
```

Comment:
this should go into the README.md

Reply:
This is meant more for development.  Most once this is done people should get it from pip.

Reply:
mhm okay